.gitignore

```diff
@@ -1,13 +1,7 @@
 __pycache__
 *.egg-info
-build
 dist
 .coverage
-.conda
-venv
```

Comment:
Emmm...Why `.conda` and `venv` are removed as well 😭? Without them my virtual env would flood the source control system.

Reply:
they are not `pymatviz`-specific. you can add `.conda` and `venv` to your global `.gitignore` which applies to all repos. you'll never want to commit those

Reply:
Got it! Thanks.